core/trino-main/src/test/java/io/trino/execution/TestCreateMaterializedViewTask.java

```diff
@@ -164,7 +164,7 @@ public void testCreateMaterializedViewIfNotExists()
                 ImmutableList.of(),
```

Comment:
Please remove unused import `io.trino.FeaturesConfig` from this class.

Reply:
done